tests/ops/functions/test_generator.py

```diff
@@ -277,32 +267,19 @@ def test_sparse_hamiltonian(self):
 
     def test_hermitian_inverse(self):
         """Test a Hermitian inverted generator is correct"""
-        gen = qml.generator(qml.adjoint(HermitianOp), format="observable")(0.5, wires=0)
-        assert gen.name == "Hermitian"
-        assert np.all(gen.parameters[0] == -HermitianOp.H)
-
         gen = qml.generator(HermitianOp(0.5, wires=0).inv(), format="observable")
         assert gen.name == "Hermitian"
         assert np.all(gen.parameters[0] == -HermitianOp.H)
 
     def test_sparse_hamiltonian_inverse(self):
         """Test a SparseHamiltonian inverted generator is correct"""
-        gen = qml.generator(qml.adjoint(SparseOp), format="observable")(0.5, wires=0)
-        assert gen.name == "SparseHamiltonian"
-        assert np.all(gen.parameters[0].toarray() == -SparseOp.H.toarray())

```

Comment:
Do I understand correctly that this is tested in the adjoint tests now?

Reply:
This deleted section tested the same as what's right below it.  

`qml.adjoint` would call `op.adjoint()` which would return a copy with in-place inversion.  This test wanted to make sure in-place inversion gave the generator a minus-sign.  That is still tested.